pkg/stages/steps_init.go

```diff
@@ -802,7 +802,7 @@ func GetKairosInitramfsFilesStage(sis values.System, l types.KairosLogger) ([]sc
 			}
 
 			// Now network
-			// we default to networmanager
+			// we default to networmanmager
```

Comment:
You managed to misspell it even further looool

Reply:
LOLOL AI is the best